### ISSUE TITLE
peer reviews

### ISSUE THREAD

**Issue by klano** [martin-fleck/cra-ttc2016#7]
Here is my feedback. Evaluation forms will be submitted by my co-author. 

Excel solution:

This has clearly produced good results, but it is not clear how it uses transformation technologies, instead of existing non-MT optimisation tools.
In addition, the heuristics in Section 3 seem to require tuning of the algorithm to specific CRA problems. The idea of creating a main class could lead to unbalanced class diagrams. Such diagrams are local but not global optima. The approach does not seem to produce optimal solutions (eg., for C, D, E cases), although the efficiency is very impressive.

Henshin solution:

An interesting investigation of GA application to the problem. It is unclear how Henshin rules are integrated into the GA - is the control of these external? Is the GA and rule control coded in Java? Some example code would be helpful. The population sizes used seem quite small. 
Section 4: 'CPA' should be 'CRA'
At some point the best results obtained per model should be given.
Section 4.2: explain the abbreviations RS, JSC, etc.

**Comment by maximiliano-vela** [comment]
Thanks for the feedback Kevin, but I don't think you were one of our opponents (Excel solution). Please check so that the other solution is not left unreviewed.

Cheers,

Maxi.

**Comment by fikovnik** [comment]
Indeed. Kevin, please have a look at the MDEOptimiser that you were supposed to review. Thanks a lot!

**Comment by fikovnik** [comment]
@klano Could you please change the title to "Kevin Lano evaluation of Excel". Thanks!
